hazelcast/generated-sources/include/hazelcast/client/protocol/codec/ProtocolCodecs.h

```diff
@@ -526,6 +526,7 @@ namespace hazelcast {
                 public:
                     static const ClientMessageType REQUEST_TYPE;
                     static const bool RETRYABLE;
+                    static const char* OPERATION_NAME;
```

Comment:
better to use constexpr and initialize in place:
`static constexpr const char * OPERATION_NAME  =  "Client.AddMembershipListener";`